### PR TITLE
[NET-0004] Add supplier and contact schema

### LINKED ISSUE #4: NET-0004: Create supplier model
**DoD**
- [x] Contact model created. Should contain relevant contact info.
- [x] Supplier model created. Should contain relevant info on suppliers.

**Suggested Schema**
Contact:
```
numbers: [string],
emails: [string],
name: string (optional, used if contact person),
```
Supplier
```
name: string,
address: [string],
contacts: [Contact],
brands: [string],
notes: string,
```

### PR DESCRIPTION
**Pull request by RayAdrian**
## Description

- Added Supplier and Contact model schemas
- Added mongoose and http-errors to package.json
- Resolves #4 

**Changelog:**
- [x] Contact model created. Should contain relevant contact info.
- [x] Supplier model created. Should contain relevant info on suppliers.

**Screenshot/s:**
Swagger
image

### CODE REVIEW

File: models/supplier.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    type: [String],
+    required: true
+  },
+  emails: {
+    type: [String],
+    required: true
+  }
+},{
+    timestamps: true
+})
```
Comment (allendomingo) [inline models/supplier.js]: This code will be reused, so it might be better to place it in a separate file

**Comment by allendomingo** [comment]
Also add swagger documentation for the model.

**Comment by RayAdrian** [comment]
Added swagger documentation 📖

**Review by allendomingo — APPROVED** [review]
Looks good to me 💯

**Pull request merged**